electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+The switch we use is a Single Pole - Single Throw switch. This means that either the two terminals are connected together in one switch position or there is a physical disconnect when the switch is in the other position. Make sure to test the switch and know which are its **ON** and **OFF** positions previous to connecting it to anything. An easy way to test the switch is to use the ’Diode’ setting on your Digital Multimeter (DMM).
+It is extremely important to plug the battery in the correct direction into the volt meter as well, as plugging it in backwards (even for an instant) could damage or destroy the volt meter.
+
+Using the Tamiya connectors, connect the battery to the switch and then the volt meter according to Figure 4. The Tamiya connectors allow you to disconnect the battery from the rest of the system so you can recharge the battery.
+
+**Table 3. Necessary Parts**
+
+Item | Ref | Qty | Image 
+---- | --- | --- | ----- 
+Battery | E36 | 1 | <p align="center"> <img src="../images/components/electronics/E36.png" width="25%"> </p>
+16 AWG Wire (Black) | X2 | 1 | <p align="center"> <img src="../images/components/wiring/X2.png" width="25%"> </p>
+16 AWG Wire (Red) | X1 | 1 |  <p align="center"> <img src="../images/components/wiring/X1.png" width="25%"> </p>
+Battery Charger | E41 | 1 | <p align="center"> <img src="../images/components/electronics/E41.png" width="25%"> </p>
+Tamiya Connectors| E35 | 1 | <p align="center"> <img src="../images/components/electronics/E35.png" width="25%"> </p>
+
+![Battery and board connections](../images/electronics/battery_wiring.png)
+
+**Figure 4. Connecting the Battery to switch and board**
+ 
+# 3. Connecting Data Cables
+**NOTE**: Before proceeding, you need to have completed the assembly of the custom PCB board and populated all the components. If you have not yet done so, build the Control Board now by following the document at /Electrical/PCB Assembly.pdf.
+
+Next, we will next plug in all the cables that allow the various devices to communicate with each other.
+
+**Table 4. Necessary Parts**
+
+Item | Ref | Qty | Image 
+---- | --- | --- | ----- 
+Rectangular 6P 1x6 Jumper Cable | E26 | 1 | <p align="center"> <img src="../images/components/electronics/E26.png" width="25%"> </p>
+USB Power Cable | E28 | 1| <p align="center"> <img src="../images/components/electronics/E28.png" width="25%"> </p>
+Rectangular 40P 2x20 Ribbon Cable | E29 | 1 | <p align="center"> <img src="../images/components/electronics/E29.png" width="25%"> </p>
+Rectangular 16P Ribbon Cable | E30 | 1| <p align="center"> <img src="../images/components/electronics/E30.png" width="25%"> </p>
+ USB to TTL Serial Adapter | E48 | 1| <p align="center"> <img src="../images/components/electronics/E48.png" width="25%"> </p>
```

Comment:
Make this table 2 columns, refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md